src/dmd/statement.h

```diff
@@ -698,6 +698,8 @@ class LabelDsymbol : public Dsymbol
 public:
     LabelStatement *statement;
 
+    bool deleted;               // set if deleted
```

Comment:
It isn't very obvious what "deleted" means in this context. Could you please elaborate in the comment?